gr-filter/grc/filter_ival_decimator.block.yml

```diff
@@ -0,0 +1,34 @@
+id: ival_decimator
+label: IValue Stream Decimator
+
+parameters:
+-   id: datatype
+    label: Input Type
+    dtype: enum
+    options: [byte, short]
+    option_attributes:
+        datatype: ['1', '2']
```

Comment:
As noted in the other comments, I'm not a fan of the hardcoded mappings here - would be nice to use existing enums, or just use the sizeof